process/coordinator/process.go

```diff
@@ -251,58 +251,40 @@ func (tc *transactionCoordinator) IsDataPreparedForProcessing(haveTime func() ti
 }
 
 // SaveTxsToStorage saves transactions from block body into storage units
-func (tc *transactionCoordinator) SaveTxsToStorage(body *block.Body) error {
+func (tc *transactionCoordinator) SaveTxsToStorage(body *block.Body) {
 	if check.IfNil(body) {
-		return nil
+		return
 	}
 
 	separatedBodies := tc.separateBodyByType(body)
 	for key, value := range separatedBodies {
-		err := tc.saveTxsToStorage(key, value)
-		if err != nil {
-			return err
-		}
+		tc.saveTxsToStorage(key, value)
 	}
 
 	for _, blockType := range tc.keysInterimProcs {
-		err := tc.saveCurrentIntermediateTxToStorage(blockType)
-		if err != nil {
-			return err
-		}
+		tc.saveCurrentIntermediateTxToStorage(blockType)
 	}
-
-	return nil
 }
 
-func (tc *transactionCoordinator) saveTxsToStorage(blockType block.Type, blockBody *block.Body) error {
+func (tc *transactionCoordinator) saveTxsToStorage(blockType block.Type, blockBody *block.Body) {
 	preproc := tc.getPreProcessor(blockType)
 	if check.IfNil(preproc) {
-		return nil
+		return
 	}
 
 	err := preproc.SaveTxsToStorage(blockBody)
```

Comment:
I think also this method could be modified easily to not return error: (just one return remained there)